### ISSUE TITLE
Progress bar problem

### ISSUE THREAD

**Issue by Yoti** [henkaku/henkaku#63]
After updating to HEN-9 last progress bar in any app never filled up.
For example, Offline Installer: ![2017-05-21-001508](https://cloud.githubusercontent.com/assets/1880088/26279451/34471bda-3dbd-11e7-9a01-1019074ab8a1.jpg)

**Comment by SilicaAndPina** [comment]
i dont think that progress bar has ever filled up. ive seen it do that since atleast henkaku r4. probably earlier too
